code/js/controllers/NetflixController.js

```diff
@@ -1,34 +1,80 @@
 ;(function() {
   "use strict";
 
-  var BaseController = require("BaseController"),
-      _ = require("lodash");
-
-  var multiSelectors = {
-    play: [".player-play-pause.play", ".button-nfplayerPlay"],
-    pause: [".player-play-pause.pause", ".button-nfplayerPause"],
-    playNext: [".player-next-episode", ".button-nfplayerNextEpisode"],
-    mute: [".player-control-button.volume", ".button-volumeLow, .button-volumeMedium, .button-volumeMax, .button-volumeMuted"],
-
-    playState: [".player-play-pause.pause",".button-nfplayerPause"],
-    song: [".player-status-main-title", ".title"]
-  };
+  var BaseController = require("BaseController");
 
   var controller = new BaseController({
-    siteName: "Netflix"
+    siteName: "Netflix",
+    play: ".button-nfplayerPlay",
+    pause: ".button-nfplayerPause",
+    playNext: ".button-nfplayerNextEpisode",
+    mute: [".button-volumeMuted", ".button-volumeLow", ".button-volumeMedium", ".button-volumeMax"],
+    video: "video",
+
+    playState: ".button-nfplayerPause",
   });
 
-  controller.checkPlayer = function() {
-    var that = this;
+  controller.setPosition = function(time) {
+    // https://stackoverflow.com/questions/9515704/insert-code-into-the-page-context-using-a-content-script/9517879#9517879
+    var code = `
+    var time = ${time};
```

Comment:
Can we move this into a separate `getNetflixPlayer` function or equivalent? Looks like it's duplicated 3x here